.travis.yml

```diff
@@ -27,30 +34,18 @@ jobs:
       script: sbt +jacoco
 
     - stage: test
-    - script: sbt +publishLocal +plugin/test +plugin/scripted
+    - script: sbt +publishLocal +plugin/test
       name: "Test plugins JDK 8"
       env: TRAVIS_JDK=8
-    - script: sbt +publishLocal +plugin/test +plugin/scripted
+    - script: sbt +publishLocal +plugin/test
       name: "Test plugins JDK11"
       env: TRAVIS_JDK=11
-    - script: cd docs && sbt validateDocs evaluateSbtFiles
-      name: "Validate docs, Evaluate sbt files JDK 8"
-      env: TRAVIS_JDK=8
-    - script: cd docs && sbt validateDocs evaluateSbtFiles
-      name: "Validate docs, Evaluate sbt files JDK 11"
-      env: TRAVIS_JDK=11
```

Comment:
Any reason for removing docs validation? 

It's good to have it before running the tests. In a stage that does some basic checks, like: mima, formatting, headers, docs. If all pass, move to testing (which take longer).

Reply:
ok, I just saw that you added a TODO list on the description that explains why this was removed. Fair enough.

Reply:
Fixed all this stuff and added it back in before tests on my latest commits. MiMA is failing (validly) now because one of the recent changes from: https://github.com/playframework/play-ebean/commit/5b588b4ae019844f12234fd2365c17c540c79253

Does that mean I need to version it to 7.0.0?


`error] play-ebean: Failed binary compatibility check against com.typesafe.play:play-ebean_2.12:6.0.0! Found 1 potential problems

[error]  * static method generateEvolutionScript(io.ebean.EbeanServer)java.lang.String in class play.db.ebean.EbeanDynamicEvolutions's type is different in current version, where it is (io.ebean.Database)java.lang.String instead of (io.ebean.EbeanServer)java.lang.String

[error]    filter with: ProblemFilters.exclude[IncompatibleMethTypeProblem]("play.db.ebean.EbeanDynamicEvolutions.generateEvolutionScript")

[error] java.lang.RuntimeException: Failed binary compatibility check against com.typesafe.play:play-ebean_2.12:6.0.0! Found 1 potential problems`

Reply:
Hmm, the best here is to keep the old method that receives `EbeanServer` and deprecate it.

If not possible, then indeed a new version number is need. I doesn't need to be 7.0.0 though, you can add breaking changes in 6.1.0. That's what is usually done in other projects. 

And of course, a very clear node in the release notes. 

If you can re-introduce the old method, you can then have a 6.0.1 (bug fixing) release.